README.md

```diff
@@ -404,7 +410,7 @@ us to download a file that does  does not exactly match the Image Repository met
 
 
 
-### 3.5: Compromise the Image repository to also serve the arbitrary package
+### 3.5: Compromise Both Repositories Simultaneously to Serve Arbitrary Package
```

Comment:
This section only includes instructions to compromise the image repository, but it does build upon the previous section so that both repositories are compromised.